dashboard/modules/reporter/reporter_agent.py

```diff
@@ -177,6 +190,25 @@ def _get_workers(self):
                 ]) for w in self._workers if w.status() != psutil.STATUS_ZOMBIE
             ]
 
+    def _get_raylet_stats(self):
+        curr_proc = psutil.Process()
+        # Here, parent is always raylet because the
+        # dashboard agent is a child of the raylet process.
```

Comment:
Let's make sure there's a test covering this so that it doesn't get accidentally broken.

Reply:
Done.